pvr.waipu/addon.xml.in

```diff
@@ -1,13 +1,13 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <addon
   id="pvr.waipu"
-  version="2.5.0"
+  version="2.5.1"
   name="waipu.tv PVR Client"
   provider-name="flubshi">
   <requires>@ADDON_DEPENDS@
-    <import addon="script.module.inputstreamhelper" version="0.4.1"/>
-    <import addon="script.module.requests" version="2.3.0"/>
-    <import addon="inputstream.adaptive" minversion="2.4.2"/>
+    <import addon="script.module.inputstreamhelper" version="0.5.2"/>
+    <import addon="script.module.requests" version="2.22.0"/>
```

Comment:
```suggestion
```

I think we can remove this dependency, since we already dropped the dirty script here: https://github.com/flubshi/pvr.waipu/pull/70